Model/Processor/ImportProcessor.php

```diff
@@ -159,4 +171,30 @@ public function transformConfigToScopeConfig($path, array $config)
 
         return $return;
     }
+
+    /**
+     * Resolve the config value if it's an environment
+     * variable reference.
+     *
+     * @throws \UnexpectedValueException
+     *
+     * @param string $value
+     * @return string|false the original string, or the resolved
+     * string if it's a reference or false on error
+     */
+    public function resolveValue($value)
+    {
+        $value = preg_replace_callback(
+            '/\{env:([^:\}\{]+?)\}/',
```

Comment:
I could not find any reference, but perhaps this string is always UTF-8 so the pattern should reflect that. Depends on what the YAML Symfony component does return (which is what I could not locate a normative reference for the character encoding, YAML itself supports more, but the Symfony component only a subset of it.